.travis.yml

```diff
@@ -2,6 +2,12 @@ language: php
 
 sudo: false
 
+addons:
+  apt:
+    packages:
+      - libxml2-utils
+
+
```

Comment:
How about removing this extra empty line here, @SpacePossum?

Reply:
:+1: good catch, I rely to much on tooling to clean up CS after me ;)